### DISCUSSION TITLE
TS101 or Pinecil V2 - Which is better build/quality overall?

### DISCUSSION THREAD

**Discussion by StudioNirin** [Ralim/IronOS#1821] [Q&A]
So I know people tend to recommend the Pinecil because it's cheaper, but in the UK/EU the pinecil is actually the more expensive option. 

I'd still buy the pinecil if it's the better choice (build quality / features), but I've heard elsewhere that the TS101 may have the better build? 

Thought I would ask here, as I'm sure some people here have tried both side by side! (And I'd be interested in the ironOS update for both platforms in the end). 

Thanks :)

**Comment by adman953** [comment]
I have both and while I can't tell a huge difference in the parts quality I can tell you that the TS101 one feels like a better build quality and seems to be more stable around temperature

**Reply by irmo-de** [reply]
I also vote for the TS101

**Reply by StudioNirin** [reply]
I dunno if I just had a bad ts101, but I'm now using a pinecil as a daily driver and it's way better.
Build quality is solid, holds temp, heats up fast, lots of high quality tip options...

The only downside is the screw to hold the tip in place, but I swapped mine for a little thumbscrew and now it's perfectly fine. 

Sold the ts101. Cheap build quality and didn't have the tip I needed. But otherwise, for normal soldering, it's perfectly fine and does the job very well. But I'd give the win to pinecil overall.
